requirements/test.txt

```diff
@@ -9,37 +9,51 @@ astroid==1.6.6            # via pylint
 backports.functools-lru-cache==1.6.1  # via astroid, isort, pylint
 billiard==3.6.3.0         # via -r requirements/base.txt, celery
 celery==4.4.6             # via -r requirements/base.txt, eox-core
+certifi==2020.6.20        # via requests
+chardet==3.0.4            # via requests
 configparser==4.0.2       # via -r requirements/base.txt, importlib-metadata, pylint
 contextlib2==0.6.0.post1  # via -r requirements/base.txt, importlib-metadata, zipp
-coverage==5.1             # via -r requirements/test.in
+coreapi==2.3.3            # via django-rest-swagger, openapi-codec
+coreschema==0.0.4         # via coreapi
+coverage==5.2             # via -r requirements/test.in
 django-crum==0.7.6        # via -r requirements/test.in
 django-fake-model==0.1.4  # via -r requirements/test.in
 django-oauth2-provider==0.2.6.1  # via -r requirements/test.in
+django-rest-swagger==2.2.0  # via -r requirements/test.in
 djangorestframework-oauth==1.1.0  # via -r requirements/test.in
-djangorestframework==3.9.4  # via -r requirements/base.txt, eox-core
+djangorestframework==3.9.4  # via -r requirements/base.txt, django-rest-swagger, eox-core
```

Comment:
I think that we need to add the constraint to use the same version used in ironwood.

Reply:
Ok I'll add the constraint

Reply:
Run `make upgrade` again in order to update this